ports/stm32/boards/WEACT_F411_BLACKPILL/pins.csv

```diff
@@ -0,0 +1,45 @@
+PA0,PA0
+PA1,PA1
+PA2,PA2
+PA3,PA3
+PA4,PA4
+PA5,PA5
+PA6,PA6
+PA7,PA7
+PA8,PA8
+PA9,PA9
+PA10,PA10
+PA11,PA11
+PA12,PA12
+PA15,PA15
+PB0,PB0
+PB1,PB1
+PB2,PB2
+PB3,PB3
+PB4,PB4
+PB5,PB5
+PB6,PB6
+PB7,PB7
+PB8,PB8
+PB9,PB9
+PB10,PB10
+PB12,PB12
+PB13,PB13
+PB14,PB14
+PB15,PB15
+PC14,PC14
+PC15,PC15
```

Comment:
I suggest to not expose these pins with board names, ie just do `,PC15` etc.

We can always add them in the future if they are needed, but we can't take them away.

See related #12676 and #13664.